test/package-sources/microshift-prel.toml

```diff
@@ -0,0 +1,9 @@
+{{- if env.Getenv "PREVIOUS_RELEASE_REPO" "" -}}
```

Comment:
Will we ever have a situation where we can't set up this package source?

Reply:
Yes, currently, when running 4.14 tests we get 4.13 from `rhocp-4.13`.

When 4.14 is released, we'll switch to `rhocp-4.14` to get newer `oc`, `crun/runc`, and probably some other deps, and ushift 4.14 as well.
This file is only used to add `mirror.openshift.com` repo, but when `rhocp-4.14` includes the version we want, it won't be added anymore to osbuild (until 4.16 and no 4.15 releases).

It's worth noting that `get_rel_version_repo.sh` checks if `rhocp-4.${minor}-for-rhel-9-x86_64-rpms` repo is enabled on the build host, so dev/configure_vm.sh also needs to be updated. So there's a bit of weird dependency going on.

One more thing: dnf reports that rhocp-4.14 exists, but when I enabled it and tried fetching the metadata I got 403. I'd like to break that dependency since osbuild repos are supposed to be independent from system's, but dynamically selecting right `rhocp-4.Y` for osbuild might be too much to fix current problem

Reply:
I need more time to think through the long-term strategy, but let's do that separately. I think this change improves the situation.